File: src/main/java/iteration2/Person.java
package iteration2;

import com.fasterxml.jackson.annotation.JsonProperty;

public class Person {

    @JsonProperty("id")
    private String id;
    @JsonProperty("name")
    private String name;

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public Person(String name, String id) {
        this.name = name;
        this.id = id;
    }
    public Person() {
    }


}

File: Person.py
class Person:
    def __init__(self, name: str, id: str):
        self.name = name
        self.id = id

    @property
    def name(self) -> str:
        return self._name

    @name.setter
    def name(self, name: str):
        self._name = name

    @property
    def id(self) -> str:
        return self._id

    @id.setter
    def id(self, id: str):
        self._id = id
